Test/CoreSDK.Test/Shared/TelemetryClientTest.cs

```diff
@@ -841,7 +841,7 @@ public void SendEventToValidateEndpoint()
 
             HttpClient client = new HttpClient();
             var result = client.PostAsync(
-                "https://dc.services.visualstudio.com/v2/validate",
+                "https://eus-breeze-in.cloudapp.net/v2/validate",
```

Comment:
That's fine, are you saying this endpoint could change in the final merge to develop?